bin/collator/src/command.rs

```diff
@@ -494,7 +493,7 @@ pub fn run() -> Result<()> {
             let _ = builder.init();
 
             let raw_wasm_blob =
-                extract_genesis_wasm(&cli.load_spec(&params.chain.clone().unwrap_or_default())?)?;
+                extract_genesis_wasm(&*cli.load_spec(&params.chain.clone().unwrap_or_default())?)?;
```

Comment:
This looks quite strange - what was the warning?

Reply:
Lint error was this
https://rust-lang.github.io/rust-clippy/master/index.html#borrowed_box

Reply:
Oh I understand, you deref the Box first +1